src/main/main.js

```diff
@@ -18,6 +19,7 @@ class Main {
     this.store = electronObjects.store;
 
     this.electronWorkingDir = require('path').dirname(require.main.filename);
+    console.log(this.electronWorkingDir);
```

Comment:
Do we want to keep this in the eventual merge?

Reply:
Will delete!